interface-definitions/include/firewall/bridge-hook-prerouting.xml.i

```diff
@@ -0,0 +1,37 @@
+<!-- include start from firewall/bridge-hook-prerouting.xml.i -->
+<node name="prerouting">
+  <properties>
+    <help>Bridge prerouting firewall</help>
+  </properties>
+  <children>
+    <node name="filter">
+      <properties>
+        <help>Bridge firewall prerouting filter</help>
```

Comment:
I think we can find better wording for these help strings, although I'm not sure offhand what a better option might be. We can keep this, if nothing else.